pkg/util/recover/recover_test.go

```diff
@@ -4,31 +4,34 @@ package recover
 // Licensed under the Apache License 2.0.
 
 import (
-	"strings"
 	"testing"
 
 	"github.com/sirupsen/logrus"
-	"github.com/sirupsen/logrus/hooks/test"
+
+	test_log "github.com/Azure/ARO-RP/test/util/log"
 )
 
 func TestPanic(t *testing.T) {
-	logger, hook := test.NewNullLogger()
-	log := logrus.NewEntry(logger)
+
+	h, log := test_log.NewCapturingLogger()
 
 	func() {
 		defer Panic(log)
 		panic("random error")
 	}()
 
-	if len(hook.Entries) != 2 {
-		t.Fatal(len(hook.Entries))
-	}
-
-	if hook.Entries[0].Message != "random error" {
-		t.Error(hook.Entries[0].Message)
+	expected := []test_log.ExpectedLogEntry{
+		{
+			Message: "random error",
+			Level:   logrus.ErrorLevel,
+		},
+		{
+			MessageRegex: `runtime\/debug\.Stack`,
```

Comment:
```suggestion
			MessageRegex: `runtime/debug\.Stack`,
```

(`/` doesn't have a special meaning in regex and therefore doesn't need the escape)

Reply:
oops, yeah